data/remnant/remnant missions.txt

```diff
@@ -940,6 +940,143 @@ mission "Remnant: Learn Sign Follow Up"
 
 
 
+mission "Remnant: Follow-up on Terminus Researchers A"
+	name "Check up on Wormhole Researchers"
+	description "Return to Bounty to check up on a group of wormhole researchers you met some time ago."
+	minor
+	source
+		government Remnant
+	destination Bounty
+	to offer
+		has "Remnant: Learning Sign 2: done"
+		has "Terminus exploration: done"
+		not "Deep: Wormhole Research: done"
+		not "Remnant: Follow-up on Terminus Researchers B: done"
+		not "Remnant: Follow-up on Terminus Researchers B: active"
+		random < 40
+	on offer
+		conversation
+			`Looking around at the Remnant spaceport, it suddenly strikes you how amazing this is. You never would have imagined that you would end up here back when you earned your pilot's license. It seems so long ago that your biggest concern was deciding whether to pay off your loan by ferrying cargo, doing courier jobs, or escorting merchants. (and on your darker days - contemplating being a pirate).`
+			branch secrets
```

Comment:
It seems to me like this branch should not exist; the current assumption of this mission is that you never followed up with the researchers, only ever retrieving the drone. (See the fact that this mission has a destination on Bounty.)